d3a_interface/schemas.py

```diff
@@ -104,13 +104,14 @@ class ScenarioSchemas:
                     "number_of_clones": {"type": "number"},
                     "uuid": {"type": "string"},
                     "libraryUUID": {"anyOf": [{"type": "string"}, {"type": "null"}]},
-                    "avg_power_W":  {"anyOf": [{"type": "number"}, {"type": "null"}]},
-                    "hrs_per_day":  {"anyOf": [{"type": "number"}, {"type": "null"}]},
+                    "avg_power_W": {"anyOf": [{"type": "number"}, {"type": "null"}]},
+                    "hrs_per_day": {"anyOf": [{"type": "number"}, {"type": "null"}]},
                     "hrs_of_day": {"anyOf": [{"type": "array"}, {"type": "null"}]},
                     "initial_buying_rate": {"type": "number"},
                     "final_buying_rate": {"anyOf": [{"type": "number"}, {"type": "null"}]},
                     "fit_to_limit": {"type": "boolean"},
-                    "update_interval":  {"anyOf": [{"type": "number"}, {"type": "null"}]},
+                    "grid_connected": {"type": "boolean"},
```

Comment:
So this is the only line that was added, right?
I am still overwhelmed by the whitespace changes. I propose to revert them.

Reply:
@hannesdiedrich Reverted